src/payload.config.ts

```diff
@@ -47,7 +47,7 @@ export default buildConfig({
         fromName: 'DCC',
         fromAddress: 'chartraj@mit.edu',
     },
-    serverURL: process.env.PAYLOAD_PUBLIC_SERVER_URL || 'http://localhost:3000',
+    serverURL: "",
```

Comment:
This may have ramifications for claim flow

Reply:
Ok, let's talk through the claim flow, how the claim page, which is also operated in the context of a single tenant knows how to reach out to the right admin dashboard domain.